_data/simple-icons.json

```diff
@@ -1223,6 +1223,11 @@
             "hex": "0066B1",
             "source": "https://www.bmw.de/"
         },
+        {
+            "title": "Boehringer Ingelheim",
+            "hex": "003366",
+            "source": "https://cd.boehringer-ingelheim.com/template/Assets/images/header/bi-logo_web_small.svg"
```

Comment:
Can you go with https://cd.boehringer-ingelheim.com for the source? We generally prefer links to pages over files to reduce the possibility of future "link rot".